src/canisters/delastic_search.mo

```diff
@@ -29,4 +45,50 @@ actor DelasticSearch {
   public func removeRecord(oldRecordId : Text, indexKeys : [Text]) : async () {
     DS.removeRecord(index, oldRecordId, indexKeys);
   };
+
+  // Example of type inserted in the index
+  type Question = {
```

Comment:
Since this is an example, let's find another spot in the repo for these changes. This file is meant to stay agnostic of how the package is implemented because the types generated from this file are used to create an npm package for JS clients to query whatever canister holds the index. I can explain more on a call

Reply:
Allright! Makes sense indeed.